src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String HORIZONTAL_LINE = "____________________________________________________________";
+    public static final String COMMAND_BYE = "bye";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_MARK = "mark";
+    public static final String COMMAND_UNMARK = "unmark";
+
+    private static TaskList TASKLIST = new TaskList();
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        greetUser();
+
+        while(true){
+            executeCommand(getUserCommand());
+        }
     }
+
+    public static void greetUser(){
```

Comment:
Good way to split print functions out! Would it be better to split them out as a separate class like Display.java? It might make your codebase appear even cleaner :)